draft-ietf-quic-transport.md

```diff
@@ -4729,18 +4729,23 @@ ECN codepoints set in duplicate packets (see {{using-ecn}}).
 
 ## Stateless Reset Oracle {#reset-oracle}
 
+Stateless resets create a possible denial of service attack analogous to a TCP
+reset injection. This attack is possible if an attacker is able to cause a
+stateless reset token to be generated for a connection with a selected
+connection ID. An attacker that can cause this token to be generated can reset
+an active connection with the same connection ID.
+
 An attacker that can cause a server to emit a stateless reset token can force
```

Comment:
I think you can remove this paragraph now.  If there's something important here I'm missing, maybe work it into an adjacent paragraph.